src/main/java/uk/gov/companieshouse/itemhandler/email/CertifiedCopyOrderConfirmation.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.companieshouse.itemhandler.email;
+
+import java.util.List;
+
+public class CertifiedCopyOrderConfirmation extends OrderConfirmation {
+    private String companyName;
+    private String companyNumber;
```

Comment:
`companyName` and `companyNumber` appear in both order confirmation types.

Reply:
Moved to super class.